src/pages/components/modal/accessibility.mdx

```diff
@@ -17,400 +17,109 @@ import {
   ListItem,
 } from '@carbon/react';
 
+<PageDescription>
+
+Design annotations are needed for specific instances shown below, but for the
+standard modal dialog component, Carbon already incorporates accessibility.
```

Comment:
Edit to add clarity, I had to read the sentence a couple of times to make sure I understood it correctly. Just a suggestion.

```suggestion
Design annotations are needed for specific instances shown below, except for the
standard modal dialog component that Carbon already incorporates accessibility.
```

Reply:
@aagonzales That's the stock pre-amble. While I agree it can get improved on, I'm not sure the rewrite improves it enough.
For most components, we're conveying that 'you're all good.' For some, we're trying to convey 1) most of the stuff is taken care of 2) in some (but depending on the component, not all) situations you need to make some design annotations.
I suggest we come back to the pre-amble with fresh eyes after the holidays and do a global update once we get something everyone thinks is an improvement :)

Reply:
Yeah that's fine to come back to this. I think its just the phrasing/sentence structure that is causing confusion for me. It would be great if we could have a content designer review the boilerplate intros and content.